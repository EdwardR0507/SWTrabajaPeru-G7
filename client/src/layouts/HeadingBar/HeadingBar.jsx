import React from "react";
import NavigateNextIcon from "@material-ui/icons/NavigateNext";
import Container from "@material-ui/core/Container";
import Typography from "@material-ui/core/Typography";
import { withStyles } from "@material-ui/core/styles";

const StyledNavigate = withStyles({
  root: {
    width: "20px",
    height: "20px",
    margin: "auto 10px",
  },
})(NavigateNextIcon);

const StyledContainer = withStyles({
  root: {
    backgroundColor: "#212121",
    width: "80%",
    height: "50px",
    padding: "15px",
    marginTop: "2em",
    marginLeft: "auto",
    marginRight: "auto",
  },
})(Container);

const StyledContainerContent = withStyles({
  root: {
    display: "flex",
    alignItems: "center",
    color: "#ffffff",
  },
})(Container);

const StyledTypography = withStyles({
  root: {
    fontStyle: "normal",
    fontWeight: "500",
    fontSize: "14px",
    lineHeight: "16px",
    letterSpacing: "1.25px",
    textTransform: "uppercase",
  },
})(Typography);

const HeadingBar = ({ before, after }) => {
  return (
    <>
      <StyledContainer>
        <StyledContainerContent>
          <StyledTypography>TRABAJADOR </StyledTypography>
          <StyledNavigate></StyledNavigate>
          <StyledTypography>MIS SERVICIOS </StyledTypography>
        </StyledContainerContent>
      </StyledContainer>
    </>
  );
};

export default HeadingBar;
