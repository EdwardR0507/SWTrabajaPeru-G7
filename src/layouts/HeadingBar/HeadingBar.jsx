import React from "react";
import NavigateNextIcon from "@material-ui/icons/NavigateNext";
import Container from "@material-ui/core/Container";
import Typography from "@material-ui/core/Typography";
import theme from "../../themes/themes";
import { withStyles } from "@material-ui/core/styles";

const StyledNavigate = withStyles({
  root: {
    width: "20px",
    height: "20px",
    margin: "auto 10px",
  },
})(NavigateNextIcon);

const StyledContainer = withStyles({
  root: {
    backgroundColor: theme.headingBar.primary.main,
    width: "80%",
    height: "50px",
    padding: "15px",
    marginTop: "2em",
    marginLeft: "auto",
    marginRight: "auto",
  },
})(Container);

const StyledContainerContent = withStyles({
  root: {
    display: "flex",
    alignItems: "center",
    color: theme.colorLetter.primary.main,
  },
})(Container);

const StyledTypography = withStyles({
  root: {
    fontStyle: "normal",
    fontWeight: "500",
    fontSize: "0.8em",
    lineHeight: "16px",
    letterSpacing: "1.25px",
    textTransform: "uppercase",
  },
})(Typography);

const HeadingBar = ({ before, after }) => {
  return (
    <>
      <StyledContainer>
        <StyledContainerContent>
          <StyledTypography>{before}</StyledTypography>
          <StyledNavigate></StyledNavigate>
          <StyledTypography>{after}</StyledTypography>
        </StyledContainerContent>
      </StyledContainer>
    </>
  );
};

export default HeadingBar;
