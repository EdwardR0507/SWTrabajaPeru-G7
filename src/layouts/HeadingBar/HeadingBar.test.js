/*Importamos las librerias principales*/
import React from "react";
import '@testing-library/jest-dom/extend-expect';
import { shallow } from "enzyme";
import HeadingBar from "./HeadingBar";
/*Renderizado del componente IHeadingBar*/
describe("HeadingBar", () => {
  it('render "HeadingBar"', () => {
    const wrapper = shallow(<HeadingBar />);
    expect(wrapper).toMatchSnapshot()
  });
});