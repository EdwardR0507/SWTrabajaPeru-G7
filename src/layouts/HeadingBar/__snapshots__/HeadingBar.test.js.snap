// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadingBar render "HeadingBar" 1`] = `ShallowWrapper {}`;
